plugins/trakt/trakt.py

```diff
@@ -78,7 +85,10 @@ def is_new_item(item, typ=typ):
                             self.echo(message)
                 else:
                     for activity in activities:
-                        self.echo(Trakt.format_activity(activity, username, activity["action"]))
+                        message = Trakt.format_activity(activity, username, activity["action"])
+                        if "movie" in activity and activity["movie"]["ids"]["trakt"] in ratings:
+                            message += " (rated it {})".format(ratings[activity["movie"]["ids"]["trakt"]])
```

Comment:
Awesome!  Just remember to fix the unit test as well (test_new_activity_both_types)